cmd/juju/controller/listcontrollers.go

```diff
@@ -147,11 +146,15 @@ func (c *listControllersCommand) refreshControllerDetails(client ControllerAcces
 			controllerModelUUID = m.UUID
 		}
 	}
-	modelStatus, err = client.ModelStatus(modelTags...)
+	modelStatus, err := client.ModelStatus(modelTags...)
 	if err != nil {
 		return err
 	}
 
+	// This is needed since return parameter may have an Error
+	// property in the later api versions, > 3.
+	newerAPI := client.BestAPIVersion() > 3
```

Comment:
again, I think you made the compatibility code in api/ so we don't need to do another check here.

Reply:
removed.